card/mod/machines/lib/javascript/decko/slotter.js.coffee

```diff
@@ -169,30 +168,19 @@ jQuery.fn.extend
     if origin && origin[0]?
       origin.reloadSlot()
 
-
   registerAsOrigin: (type, slot) ->
     if slot.hasClass("_modal-slot")
       slot = slot.find(".modal-body .card-slot")
     slot.attr("data-#{type}-origin-slot-id", @closest(".card-slot").data("slot-id"))
-    #@closest(".card-slot").addClass("_#{type}-origin")
 
   updateSlot: (data, mode) ->
-    notice = @attr('notify-success')
     mode ||= "replace"
-    newslot = @setSlotContent data, mode, $(this)
-
-    if newslot.jquery # sometimes response is plaintext
```

Comment:
I don't see this test anywhere.  Are we assuming we no longer do this?

Reply:
`setSlotContent` checks if data is a string. In that case we don't do this.